messages/messages.json

```diff
@@ -29,5 +29,8 @@
   "noPromptAuth": "do not prompt for auth confirmation in demo mode",
   "disableMasking": "disable masking of user input (for use with problematic terminals)",
   "clientSecretStdin": "OAuth client secret of personal connected app?",
-  "invalidInstanceUrl": "Invalid instance URL. Specify a Salesforce instance URL using the format <domainname>.salesforce.com"
+  "invalidInstanceUrl": "Invalid instance URL. Specify a Salesforce instance URL using the format <domainname>.salesforce.com",
+  "accessTokenStdin": "Access token of user to use for authentication",
+  "noPrompt": "do not prompt for confirmation",
```

Comment:
Optional edit:  (CCX style is to use contractions, although I know that many of the CLI messages don't.  But I think it looks better.)

"don't prompt for confirmation"

Reply:
@jshackell-sfdc I am reusing an existing message here. If I change it, it will affect the other commands as well.

Reply:
Okay, fine to leave it.